pkg/crc/segment/segment.go

```diff
@@ -58,6 +58,10 @@ func (c *Client) Close() error {
 }
 
 func (c *Client) Upload(action string, duration time.Duration, err error) error {
+	if version.IsOkdBuild() {
+		return nil
```

Comment:
Are we sure about this (cc @sspeiche), Our motive is crc usage collection, doesn't matter if it is okd or ocp right?

Reply:
Some arguments:
* Community doesn't have access to Segment. They can't look at statistics. 
* The ID we used is also something Red Hat related. 
* I would expect community builds to be free from all sort of external services.

Reply:
Agree we should try to keep out off community bits, though would be interesting to know how many OKD instances there are if we can...I think we just need to be transparent we are doing it and allow for opt-out.

Reply:
@sspeiche As of now by default it is opt out, it is upto the user to enable it or not.

Reply:
OK, I removed this from the PR. OKD will send events if turned on.